tests/torch/test_sanity_third_party.py

```diff
@@ -260,9 +262,10 @@ def test_install_mmdet(self, pip_cache_dir):
         pip_runner.run_pip("install mmcv-full==1.2.0 "
                            "-f https://download.openmmlab.com/mmcv/dist/cu102/torch{}/index.html".format(
             BKC_TORCH_VERSION),
-                           cwd=self.MMDET_PATH)
+                           cwd=self.MMDET_PATH,
+                           use_cache=False)
         pip_runner.run_pip("install onnx onnxruntime", cwd=self.MMDET_PATH)
-        pip_runner.run_pip("install torchvision", cwd=self.MMDET_PATH)
+        pip_runner.run_pip("install torchvision==0.9.1", cwd=self.MMDET_PATH)
```

Comment:
Should we set up a constant for this in nncf/version.py, along with `BKC_TORCH_VERSION`?

Reply:
Done